<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ANC_UI</class>
 <widget class="QWidget" name="ANC_UI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>363</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>294</width>
    <height>363</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="axis_groupBox_2">
       <property name="maximumSize">
        <size>
         <width>274</width>
         <height>343</height>
        </size>
       </property>
       <property name="title">
        <string>Attocube ANC150 Control Panel</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0" colspan="2">
         <widget class="QGroupBox" name="axis_groupBox">
          <property name="title">
           <string>Axis Controls</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QLabel" name="axis_label">
             <property name="text">
              <string>Axis</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="axis_comboBox"/>
           </item>
           <item row="0" column="2">
            <widget class="QComboBox" name="axis_mode_comboBox"/>
           </item>
           <item row="0" column="3">
            <widget class="QLabel" name="axis_mode_label">
             <property name="text">
              <string>Mode</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QGroupBox" name="move_groupBox">
          <property name="title">
           <string>Move</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="move_dir_label">
             <property name="text">
              <string>Direction</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2" colspan="2">
            <widget class="QLabel" name="move_step_label">
             <property name="text">
              <string>Steps</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="move_mode_label">
             <property name="text">
              <string>Mode</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QComboBox" name="move_dir_comboBox"/>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="move_mode_comboBox"/>
           </item>
           <item row="1" column="2" colspan="2">
            <widget class="QDoubleSpinBox" name="move_step_doubleSpinBox"/>
           </item>
           <item row="2" column="1" colspan="2">
            <widget class="QPushButton" name="move_start_pushButton">
             <property name="text">
              <string>Start</string>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QPushButton" name="move_stop_pushButton">
             <property name="text">
              <string>Stop</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QDoubleSpinBox" name="frequency_doubleSpinBox"/>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="frequency_label">
          <property name="text">
           <string>Frequency</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QDoubleSpinBox" name="voltage_doubleSpinBox"/>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="voltage_label">
          <property name="text">
           <string>Voltage</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QDoubleSpinBox" name="cap_doubleSpinBox"/>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="cap_label">
          <property name="text">
           <string>Capacitance</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="2">
         <widget class="QGroupBox" name="patternnumber_groupBox">
          <property name="title">
           <string>Pattern Number</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="pn_dir_label">
             <property name="text">
              <string>Direction</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="pn_num_label">
             <property name="text">
              <string>Number</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QComboBox" name="pn_dir_comboBox"/>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="pn_num_comboBox"/>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="pn_apply_pushButton">
             <property name="text">
              <string>Apply</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
